protocol-units/da/movement/protocol/light-node/src/passthrough.rs

```diff
@@ -35,7 +35,8 @@ where
 		+ Serialize
 		+ for<'de> Deserialize<'de>
 		+ Clone
-		+ 'static,
+		+ 'static
+		+ std::fmt::Debug,
```

Comment:
What's the cause of this `Debug` bound pollution everywhere?

Reply:
To debug DA block content, we need to add the constraint on C everywhere. DaBlob<C> depend on C.

Reply:
For test and event to debug to be able to see DA block is important.